core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -89,6 +90,8 @@
     private Runnable requestTimeoutHandler;
     private long maxRequestLength;
 
+    private boolean timedOut;
```

Comment:
I think we still need to keep this in the super class because the users of the deprecated method will not want to see a breaking change.

Reply:
Removed it. Thanks!